assets/xml/objects/object_ob.xml

```diff
@@ -1,84 +1,100 @@
 ﻿<Root>
     <File Name="object_ob" Segment="6">
```

Comment:
Since you completely documented this object, can you include a description of the file above the \<File\> node? You can look at something like `object_giant.xml` for examples.